Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
-import './Card.css';
+import './Card.scss';
+
+const CardBanner = () => {
+    return (
+        <img src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" alt="react logo"></img>
```

Comment:
*img* is a self-closing tag. This is fine, but you can also just do <img src="imgSrc" alt="imgAlt" />